docs/aws.md

```diff
@@ -286,7 +286,8 @@ export KOPS_STATE_STORE=s3://prefix-example-com-state-store
 ```
 
 Note: You don’t have to use environmental variables here. You can always define
-the values using the –name and –state flags later.
+the values using the –name and –state flags later. Alternately, you may define 
+[config file](config_file.md)
```

Comment:
`Alternately, you may define a [config file](config_file.md).`
("a" and . at the end)